libs/scripts/taxonomy.js

```diff
@@ -84,23 +84,19 @@ function parseTaxonomyJson(data, root) {
         : LEVEL_INDEX.level1);
 
     const name = level3 || level2 || level1;
-
     const category = row[TAXONOMY_FIELDS.type]?.trim().toLowerCase() || INTERNALS;
 
     // skip duplicates
     if (!isProduct(category) && taxonomy.topics[name]) return taxonomy;
     if (isProduct(category) && taxonomy.products[name]) return taxonomy;
```

Comment:
may as well change this to
```suggestion
    if ((!isProduct(category) && taxonomy.topics[name])
      || (isProduct(category) && taxonomy.products[name])
    ) return taxonomy;
```